spec/features/job_seekers_can_view_vacancies_spec.rb

```diff
@@ -11,20 +11,25 @@
     expect(page).to have_selector('.vacancy', count: Vacancy.default_per_page)
   end
 
-  scenario 'Only published, non-expired vacancies are visible in the list', elasticsearch: true do
+  scenario 'Only vacancies satisfying all publishing conditions are listed', elasticsearch: true do
     valid_vacancy = create(:vacancy)
 
     expired = build(:vacancy, :expired)
     expired.send :set_slug
-    expired.save(validete: false)
```

Comment:
The fact this test passed with this in highlights that it may very well be redundant?